### PR TITLE
Formatting code homework

### PR DESCRIPTION
**Pull request by dementtor**
Here is my work

### CODE REVIEW

File: formatting-code/JavaScript-code.js [hunk 1]
```diff
@@ -1,19 +1,109 @@
...
+        pX = event.x - 5;
+        pY = event.y;
+    }
+
+    if (b == "Netscape") {
+        if (document.layers['ToolTip'].visibility == 'show') {
+            PopTip();
+        }
+    } 
+	else {
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:29]: I think shouldn't have tab space for line #29.

File: formatting-code/JavaScript-code.js [hunk 2]
```diff
@@ -1,19 +1,109 @@
...
+document.onmousemove = mouseMove;
+if (b == "Netscape"){
+    document.captureEvents(Event.MOUSEMOVE);
+}
+function mouseMove(evn){
+    if (b == "Netscape"){
+        pX = evn.pageX - 5;
+        pY = evn.pageY;
+    } 
+	else{
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:19]: I think shouldn't have tab space for line #19.

File: formatting-code/JavaScript-code.js [hunk 3]
```diff
@@ -1,19 +1,109 @@
...
+if ((navigator.userAgent.indexOf('MSIE 5') > 0) || (navigator.userAgent.indexOf('MSIE 6')) > 0){
+    addScroll = true;
+}
+var off = 0;
+var txt = "";
+var pX = 0;
+var pY = 0;
+
+document.onmousemove = mouseMove;
+if (b == "Netscape"){
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:11]: Empty line for line #11 might be better.

File: formatting-code/JavaScript-code.js [hunk 4]
```diff
@@ -1,19 +1,109 @@
...
-		}	}}function HideTip() {	args=HideTip.arguments;
-	if (b == "Netscape") 		{ document.layers['ToolTip'].visibility='hide'; }
-	else 		{ document.all['ToolTip'].style.visibility='hidden'; }
-}function HideMenu1() {	if (b == "Netscape") 		{ document.layers['menu1'].visibility='hide'; } 	else  		{ document.all['menu1'].style.visibility='hidden'; }}
-function ShowMenu1(){if (b == "Netscape")    {    theLayer = eval('document.layers[\'menu1\']');	    theLayer.visibility='show';	   } else {    theLayer = eval('document.all[\'menu1\']');    theLayer.style.visibility='visible';   }}//
-function HideMenu2()   			{	     if (b == "Netscape") 		{ document.layers['menu2'].visibility='hide'; }	else 
-		{ document.all['menu2'].style.visibility='hidden'; } } function ShowMenu2(){if (b == "Netscape")    {    theLayer = eval('document.layers[\'menu2\']');	    theLayer.visibility='show';	   } else {    theLayer = eval('document.all[\'menu2\']');    theLayer.style.visibility='visible';   }} // fostata
\ No newline at end of file
+b = navigator.appName.addScroll = false;
+if ((navigator.userAgent.indexOf('MSIE 5') > 0) || (navigator.userAgent.indexOf('MSIE 6')) > 0){
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:2]: Empty line for line #2 might be better.

File: formatting-code/JavaScript-code.js [hunk 5]
```diff
@@ -1,19 +1,109 @@
...
-		theLayer.style.pixelTop=pY+15;		theLayer.style.visibility='visible';
-		}	}}function HideTip() {	args=HideTip.arguments;
-	if (b == "Netscape") 		{ document.layers['ToolTip'].visibility='hide'; }
-	else 		{ document.all['ToolTip'].style.visibility='hidden'; }
-}function HideMenu1() {	if (b == "Netscape") 		{ document.layers['menu1'].visibility='hide'; } 	else  		{ document.all['menu1'].style.visibility='hidden'; }}
-function ShowMenu1(){if (b == "Netscape")    {    theLayer = eval('document.layers[\'menu1\']');	    theLayer.visibility='show';	   } else {    theLayer = eval('document.all[\'menu1\']');    theLayer.style.visibility='visible';   }}//
-function HideMenu2()   			{	     if (b == "Netscape") 		{ document.layers['menu2'].visibility='hide'; }	else 
-		{ document.all['menu2'].style.visibility='hidden'; } } function ShowMenu2(){if (b == "Netscape")    {    theLayer = eval('document.layers[\'menu2\']');	    theLayer.visibility='show';	   } else {    theLayer = eval('document.all[\'menu2\']');    theLayer.style.visibility='visible';   }} // fostata
\ No newline at end of file
+b = navigator.appName.addScroll = false;
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:1]: In my opinion should have 'var' statement in begining.

File: formatting-code/JavaScript-code.js [hunk 6]
```diff
@@ -1,19 +1,109 @@
...
+if ((navigator.userAgent.indexOf('MSIE 5') > 0) || (navigator.userAgent.indexOf('MSIE 6')) > 0){
+    addScroll = true;
+}
+var off = 0;
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:5]: Empty line for line #5 might be better.

File: formatting-code/JavaScript-code.js [hunk 7]
```diff
@@ -1,19 +1,109 @@
...
+var txt = "";
+var pX = 0;
+var pY = 0;
+
+document.onmousemove = mouseMove;
+if (b == "Netscape"){
+    document.captureEvents(Event.MOUSEMOVE);
+}
+function mouseMove(evn){
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:14]: Empty line for line #14 might be better.

File: formatting-code/JavaScript-code.js [hunk 8]
```diff
@@ -1,19 +1,109 @@
...
+            PopTip();
+        }
+    } 
+	else {
+        if (document.all['ToolTip'].style.visibility == 'visible') {
+            PopTip();
+        }
+    }
+}
+function PopTip() {
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:35]: Empty line for line #35 might be better.

File: formatting-code/JavaScript-code.js [hunk 9]
```diff
@@ -1,19 +1,109 @@
...
+        theLayer = eval('document.layers[\'ToolTip\']');
+        if ((pX + 120) > window.innerWidth)
+        {
+            pX = window.innerWidth - 150;
+        }
+        theLayer.left = pX + 10;
+        theLayer.top = pY + 15;
+        theLayer.visibility = 'show';
+    } 
+	else {
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:46]: I think shouldn't have tab space for line #46.

File: formatting-code/JavaScript-code.js [hunk 10]
```diff
@@ -1,19 +1,109 @@
...
+        theLayer = eval('document.all[\'ToolTip\']');
+        if (theLayer) {
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:48]: Empty line for line #48 might be better.

File: formatting-code/JavaScript-code.js [hunk 11]
```diff
@@ -1,19 +1,109 @@
...
+            pX = event.x - 5;
+            pY = event.y;
+            if (addScroll) {
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:51]: Same thing

File: formatting-code/JavaScript-code.js [hunk 12]
```diff
@@ -1,19 +1,109 @@
...
+                pX = pX + document.body.scrollLeft;
+                pY = pY + document.body.scrollTop;
+            }
+            if ((pX + 120) > document.body.clientWidth)
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:55]: !

File: formatting-code/JavaScript-code.js [hunk 13]
```diff
@@ -1,19 +1,109 @@
...
+            {
+                pX = pX - 150;
+            }
+            theLayer.style.pixelLeft = pX + 10;
+            theLayer.style.pixelTop = pY + 15;
+            theLayer.style.visibility = 'visible';
+        }
+    }
+}
+function HideTip() {
```
Comment (anaseya) [inline formatting-code/JavaScript-code.js:65]: !

File: formatting-code/Manager.php [hunk 14]
```diff
@@ -1,45 +1,53 @@
-<?php namespace League\CLImate\Settings;
+<?php 
+
```
Comment (anaseya) [inline formatting-code/Manager.php:2]: Why empty line?

File: formatting-code/Manager.php [hunk 15]
```diff
@@ -1,45 +1,53 @@
...
+	}
 
- {
-            $this->$method($setting);}
+    public function importSetting($setting){
+        $short_name = basename(str_replace('\\', '/', get_class($setting))); 
+		$method = 'importSetting' . $short_name;
 
+		if (method_exists($this, $method)) {
+            $this->$method($setting);
```
Comment (anaseya) [inline formatting-code/Manager.php:50]: Where is the tab space?

File: formatting-code/Manager.php [hunk 16]
```diff
@@ -1,45 +1,53 @@
...
-        $short_name = basename(str_replace('\\', '/', get_class($setting))); $method = 'importSetting' 
-		. $short_name;
-if (method_exists($this, $method))
+	public function settings(){
+		return [];
+	}
 
- {
-            $this->$method($setting);}
+    public function importSetting($setting){
```
Comment (anaseya) [inline formatting-code/Manager.php:45]: importSetting($setting) { [just add space right after curly bracket]

File: formatting-code/Manager.php [hunk 17]
```diff
@@ -1,45 +1,53 @@
...
+		return ucwords(str_replace('add_', '', $name));
+	}
 
-    public function importSetting($setting)
-    {
-        $short_name = basename(str_replace('\\', '/', get_class($setting))); $method = 'importSetting' 
-		. $short_name;
-if (method_exists($this, $method))
+	public function settings(){
+		return [];
```
Comment (anaseya) [inline formatting-code/Manager.php:42]: I see here two tab spaces for the method and more two tab spaces for its body - both are wrong.

File: formatting-code/Manager.php [hunk 18]
```diff
@@ -1,45 +1,53 @@
...
-    protected function getPath($name) {       return '\\League\CLImate\\Settings\\' . $this->getClassName($name);}
-
-    protected function getClassName($name)
-    {return ucwords(str_replace('add_', '', $name));}
+    protected function getPath($name) {       
+		return '\\League\CLImate\\Settings\\' . $this->getClassName($name);
+	}
 
-	public function settings(){return [];}
+    protected function getClassName($name) {
```
Comment (anaseya) [inline formatting-code/Manager.php:37]: Might be better if all protected/private method are below public ones as is recommended for interfaces etc (see PSR recommondations for more info :P).

File: formatting-code/Manager.php [hunk 19]
```diff
@@ -1,45 +1,53 @@
...
+			return $this->settings[$key]; 
+		}
         return false;
     }
 
-    protected function getPath($name) {       return '\\League\CLImate\\Settings\\' . $this->getClassName($name);}
-
-    protected function getClassName($name)
-    {return ucwords(str_replace('add_', '', $name));}
+    protected function getPath($name) {
```
Comment (anaseya) [inline formatting-code/Manager.php:33]: Might be better if all protected/private method are below public ones as is recommended for interfaces etc (see PSR recommondations for more info :P).

File: formatting-code/Manager.php (hunk 14 above)
Comment (victorlazov) [inline formatting-code/Manager.php:2]: Druapl 7 indent for the whole file is not applied.

File: formatting-code/Manager.php (hunk 16 above)
Comment (victorlazov) [inline formatting-code/Manager.php:45]: Before also.

**Pull request merged**